src/20_Components/amp-analytics.html

```diff
@@ -124,6 +127,38 @@
   -->
   <amp-img id="cat-image-id" src="/img/cat-looking-up-300x200.jpg" width="300" height="200" attribution="visualhunt" alt="a cat" layout="responsive"></amp-img>
 
+  <!--
+    This sample fires an analytics request when the image becomes visible as a result of an action. 
+      -->
+      <div>
+          <button class="ampstart-btn caps m1" on="tap:an-image.toggleVisibility">
+             Toggle image
+          </button>
+          <amp-img hidden id="an-image" src="/img/amp.jpg" width="1080" height="610" layout="responsive" alt="AMP">
+          </amp-img>
+      </div>
+      <!--
+    The `elementview` event is issued when the image becomes visible -->
+  <amp-analytics>
+      <script type="application/json">
+      {
+        "requests": {
+          "elementview": "https://amp-publisher-samples-staging.herokuapp.com/amp-analytics/ping?user=[[.]]&account=ampbyexample&event=${eventId}"
+        },
+        "triggers": {
+          "defaultPageview": {
+            "on": "visible",
+            "request": "elementview",
+            "selector": "#an-image",
+            "vars": {
+              "eventId": "element-ui"
```

Comment:
Can you specify a more meaningful event id, one that makes it clear in the dashboard which event occured,